footprint: [[-0.35, -0.25], [-0.35, 0.25], [0.35, 0.25], [0.35, -0.25]]
footprint_padding: 0.001

robot_base_frame: base_link
update_frequency: 4.0
publish_frequency: 3.0
transform_tolerance: 0.5

resolution: 0.05

obstacle_range: 2.5
raytrace_range: 3.0


#layer definitions
static:
    map_topic: /map
    subscribe_to_updates: true

obstacles:
    observation_sources: laser rear_laser front_camera rear_camera
    laser: {data_type: LaserScan, clearing: true, marking: true, topic: scan, inf_is_valid: true}
    rear_laser: {data_type: LaserScan, clearing: true, marking: true, topic: scan_rear, inf_is_valid: true}
    front_camera: {data_type: LaserScan, clearing: true, marking: true, topic: front_camera_scan, inf_is_valid: true}
    rear_camera: {data_type: LaserScan, clearing: true, marking: true, topic: rear_camera_scan, inf_is_valid: true}

#costmap radius
#inflation:
#    inflation_radius: 0.2
